### PR TITLE
[Feature] CON 81 - Enable text wrap passages

### PR DESCRIPTION
**Pull request by habemuscode**
**Related to this issue**

https://oat-sa.atlassian.net/browse/CON-81

**Other related PRs**

- https://github.com/oat-sa/tao-core-ui-fe/pull/217
- https://github.com/oat-sa/extension-tao-mediamanager/pull/240

**Story**

As the author, I want to be able to wrap my text around pictures, so that I can produce passages that look like real-life media content.

Currently, I am only able to anchor images AS a character. In order for the text to be able to flow around it, I will need to be able to anchor it TO a character and specify how text is supposed to wrap around it

Required wrap modes:

- Float off
- Float right
- Float left

**Acceptance Criteria**

- Anchoring an image as a character should still be possible
- Text wrapping should not be available when the image is anchored as a character
- Images maintain position within their paragraph when text is added before or after
- Responsive images are still supported if the image is anchored to a character
- Exported items will contain the relevant CSS to ensure the rendering will be correct in any environment
- Exporting and reimporting an item will maintain the anchoring and wrapping

### CODE REVIEW

**Review by pnal — APPROVED** [review]
Nothing suspicious, LGTM
- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x] New code is respecting code style rules
- [x] New code is respecting best practices
- [x] New code is not subject to concurrency issues (if applicable)
- [ ] Feature is working correctly on my local machine (if applicable)
- [ ] Acceptance criteria are respected
- [ ] Pull request title and description are meaningful

**Review by gabrielfs7 — APPROVED** [review]
Code LGTM 

- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [ ] New code is respecting code style rules
- [x] New code is respecting best practices
- [x] New code is not subject to concurrency issues (if applicable)
- [ ] Feature is working correctly on my local machine (if applicable)
- [ ] Acceptance criteria are respected
- [x] Pull request title and description are meaningful

**Review by bziondik — CHANGES_REQUESTED** [review]
Update version

**Review by mike-ionut-mihai-sandu-tao — CHANGES_REQUESTED** [review]
manifest cage required

**Comment by habemuscode** [comment]
@bziondik  @mike-ionut-mihai-sandu-tao Manifest pushed

**Review by shpran — APPROVED** [review]
- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x] New code is respecting code style rules
- [x] New code is respecting best practices
- [x] New code is not subject to concurrency issues (if applicable)
- [ ] Feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [x] Pull request title and description are meaningful

**Review by bziondik — APPROVED** [review]
- [ ]  New code is covered by tests (if applicable)
- [ ]  Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x]  New code is respecting code style rules
- [x]  New code is respecting best practices
- [x]  New code is not subject to concurrency issues (if applicable)
- [ ]  Feature is working correctly on my local machine (if applicable)
- [x]  Acceptance criteria are respected
- [x]  Pull request title and description are meaningful

**Comment by gabrielfs7** [comment]
We need to check the build, probably missing some composer.json dependencies

File: views/js/qtiCreator/widgets/static/helpers/inline.js [hunk 1]
```diff
@@ -65,18 +65,22 @@ define([
                 elt = widget.element;
 
             //remove class
-            $container.removeClass('rgt lft');
+            $container.removeClass('rgt txt-rgt lft txt-lft');
```
Comment (bziondik) [inline views/js/qtiCreator/widgets/static/helpers/inline.js:68]: Do we need this? in Item Authoring we don't have option Position for images

**Review by 404passionFruit — APPROVED** [review]
- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x] New code is respecting code style rules
- [x] New code is respecting best practices
- [ ] New code is not subject to concurrency issues (if applicable)
- [ ] Feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [x] Pull request title and description are meaningful

**Comment by 404passionFruit** [comment]
Converted to draft as there is an alternative solution https://github.com/oat-sa/extension-tao-itemqti/pull/1761

**Comment by habemuscode** [comment]
Closed in favor of https://github.com/oat-sa/extension-tao-itemqti/pull/1761

**Pull request closed without merging**